packages/keystone/fetch.js

```diff
@@ -15,6 +15,12 @@ const logger = getLogger('fetch')
 const FETCH_COUNT_METRIC_NAME = 'fetch.count'
 const FETCH_TIME_METRIC_NAME = 'fetch.time'
 
+/**
+ * Should be: { [hostname: str]:[x-target: str] }
+ * @example { v1.condo.ai: 'example-app' }
+ */
+const FETCH_X_TARGET_CONFIG = JSON.parse(conf.FETCH_X_TARGET_CONFIG || '{}')
```

Comment:
It should probably the same for all domains, or have default option

Reply:
Motivation is that I want to go to all internal services (even for newly created one) with specific header by default

Reply:
Agreed, wait a day or two